test/testchunkingng.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ *    This software is in the public domain, furnished "as is", without technical
+ *    support, and with no warranty, express or implied, as to its usefulness for
+ *    any purpose.
+ *
+ */
+
+#include <QtTest>
+#include "syncenginetestutils.h"
+#include <syncengine.h>
+
+using namespace OCC;
+
+class TestChunkingNG : public QObject
```

Comment:
The cleanup is there if the test does not crash.  But it seems that the test is crashing.   need to investigate why.